react/components/AvailabilitySubscriber/index.js

```diff
@@ -63,16 +67,70 @@ class AvailabilitySubscriber extends Component {
     }
   }
 
-  handleClick = () => {
-    /* TODO @lucasecdb: we don't have a definition for which api to use yet */
-
+  handleSubmit = (e, client) => {
+    e.preventDefault()
+    const variables = {
+      acronym: 'AvailabilitySubscription',
```

Comment:
This must be pass as a param of the component.

Reply:
I think that the Entity must be fixed because that way we can create a app on the Admin that reads this information, if this data is dynamic, will be hard to create a backend and a app to manage the subscriptions.

Reply:
Ok, make senses. Lastly, an acronym is more like AS, or AVS, not the entire entity name. Likewise the profile and address that are CL and AD.